configs/openmrs/initializer_config/relationshiptypes/relationshiptypes.csv

```diff
@@ -0,0 +1,12 @@
+Uuid,Void/Retire,Name,Description,Weight,Preferred,A is to b,B is to a
+3ac37b90-6276-4f69-ab70-33128040fa76,,Parent,Parent,,,Parent,Child
+7a475706-0f57-4ed2-ae4a-04a9989fd5df,,Sibling (Frère ou soeur),Sibling (Frère ou soeur),,,Sibling (Frère ou soeur),Sibling (Frère ou soeur)
+2a76a46f-ca46-4629-bed1-8dc7aaf716a4,,Friend (Amis),Friend (Amis),,,Friend (Amis),Friend (Amis)
+63923579-8593-4852-9be2-c339f18a4aca,,Partner/Spouse (Partenaire ou conjoint),Partner/Spouse (Partenaire ou conjoint),,,Partner/Spouse (Partenaire ou conjoint),Partner/Spouse (Partenaire ou conjoint)
+c86d9979-b8ac-4d8c-85cf-cc04e7f16315,TRUE,,,,,,
+3982f469-cedc-4b2d-91ea-fe38f881e1a0,TRUE,,,,,,
+4f6b4fc4-66e5-4364-be1d-d5872ff8817e,TRUE,,,,,,
+53d8a8f3-0084-4a52-8666-c655f5bd2689,TRUE,,,,,,
+117ac27a-7fec-4834-8fad-d967c702c15a,TRUE,,,,,,
+8c8bbfe3-8454-41f9-a180-55f7058af9bd,TRUE,,,,,,
+057de23f-3d9c-4314-9391-4452970739c6,TRUE,,,,,,
```

Comment:
Are these not brought in by the excluded file below `relationshiptypes-core_demo.csv`?

Reply:
They dont seem to be excluded when I rebuilt the instance. you had them still appear

[Screencast from 24-10-24 13:10:18.webm](https://github.com/user-attachments/assets/de678a07-6336-4a82-a9d0-03fbfe2cf499)

Reply:
Let do this to be sure of what's being retired, 1.) Spine an empty instance without the file `relationshiptypes-core_demo.csv`, grab the `uuid`s for the relationships that show up and then use those when retiring. That way, we'll be sure we are retiring relationships that are not part of the `relationshiptypes-core_demo.csv` configuration.